tests/ops/integration_tests/saas/request_override/test_mailchimp_override_task.py

```diff
@@ -104,40 +104,6 @@ async def test_mailchimp_override_access_request_task(
     # links
     assert v[f"{dataset_name}:member"][0]["email_address"] == mailchimp_identity_email
 
-    logs = (
```

Comment:
was this causing false negatives? don't think it's super important, and i could see it being a bit fragile, but i think the intention was to provide some more comprehensive test coverage on the override functionality, so just curious if there's a particular reason we're removing it now

Reply:
I did it more for consistency, I agree this gives us a bit more test coverage but I wanted all of the tests to have this or none of them. I was planning on bringing this back once we create our auto-generated test runner.